sympy/combinatorics/fp_groups.py

```diff
@@ -494,6 +496,97 @@ def elements(self):
         P, T = self._to_perm_group()
         return T.invert(P._elements)
 
+    @property
+    def is_polycyclic(self):
+        if self.is_solvable:
+            return True
+        return None
+
+    def is_subgroup(self, other):
+        """
+        Return ``True`` if all elements of ``self`` belong to ``other``.
+        """
+        _grp1 = self._to_perm_group()[0]
+        _other = self._to_perm_group()[0]
+        return _grp1.is_subgroup(_other, strict=False)
+
+    def compute_polycyclic_series(self):
+        '''
+        ToDo - Update docstring.
+        '''
+        if not self.is_polycyclic:
+            raise NotImplementedError("Polycyclic methods are only"
+                                        "implemented for FpGroups")
+        # Compute the derived series.
+        derived_series = self.derived_series()
+        derived_series.insert(0, self)
+        len_derived_series = len(derived_series)
+
+        # Compute the polycyclic series by computing the
+        # subnormal series for every quotient G_i/G_i+1
+        # in the derived series and finally,
+        # lifting them to the original derived series.
+        pc_series = [derived_series[0]]
+        for i in range(0, len_derived_series-1):
+            # Compute the quotient group G_i/Gi+1.
+            q_grp = subgroup_quotient(FpSubgroup(derived_series[i]), derived_series[i+1])
+            # Return a subnormal series.
+            _subnormal_series = _subnormal_series(q_grp)
+            # The images of every subgroup in the quotient map
+            # can be lifted to the orginal group
+            gens = q_grp.generators
+            q_map = homomorphism(derived_series[i], q_grp, gens, gens)
+            for sub_group in _subnormal_series:
+                # Pre-image of subgroup in quotient map.
+                _sub_group = q_map.invert_subgroup(sub_group)
+                pc_series.append(_sub_group)
+            pc_series.append(derived_series[i+1])
+
+        return pc_series
+
+
+def subgroup_quotient(G, H):
+    '''
+    Compute the quotient group G/H
+    where, H is the subgroup of G.
```

Comment:
There shouldn't be a comma after `where`.

Reply:
There still shouldn't be a comma. Also, the docstring should explain what the arguments are for. I think that `G` and `H` should come as the first 2 arguments and the other things should be keyword arguments. Is `free_group` ever different from `fp_grp.free_group`?